core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -24,6 +24,7 @@
 
 package io.questdb.griffin;
 
+import io.questdb.PropertyKey;
```

Comment:
This import seems to be unused - the same applies to other classes changed by the PR. If you're using IDEA, please run the Optimize imports action.

Reply:
Tried to optimize all imports. But Optimize imports action over `src/` dir yield a lot changes in other files for me - so I rolled them back and leaved only files changes in my PR